### ISSUE TITLE
Getting rewards always 1 computed in utils/logits_processor.py file

### ISSUE THREAD

**Issue by RadiantCrystal** [r-three/RAD#4]
Hi Authors,

I am reimplementing your code for my project work. I was going through your codebase and algorithm. Could you please refer me to the code where you implemented the line number 6 in Algorithm? 
**Actually I am always getting all 1 rewards for my experiments. Seems like I am missing something. **
**6: pt ← softmax(zt + βρt ) // compute reweighted distribution**
